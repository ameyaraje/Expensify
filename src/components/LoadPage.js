import React from 'react';

const LoadPage = () => {
    return (
        <div className="loader">
            <img className="loader__image" src="/images/loader.gif" />
        </div>
    );
};

export default LoadPage;