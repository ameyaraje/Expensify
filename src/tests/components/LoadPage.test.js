import React from 'react';
import { shallow } from 'enzyme';

import LoadPage from '../../components/LoadPage';

test('test should render LoadPage', () => {
    const wrapper = shallow(<LoadPage />);
    
    expect(wrapper).toMatchSnapshot();
});