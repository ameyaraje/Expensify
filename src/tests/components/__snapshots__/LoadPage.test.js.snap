// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test should render LoadPage 1`] = `
<div
  className="loader"
>
  <img
    className="loader__image"
    src="/images/loader.gif"
  />
</div>
`;
